src/components/Avatar/Avatar.types.ts

```diff
@@ -67,6 +67,11 @@ export interface StatusItemsProps extends IconProps {
    * Class for status item wrapper
    */
   wrapperClassName?: string;
+  /**
+   * Place text after the icon if true
+   * @default false
+   */
+  placeTextAfterIcon?: boolean;
```

Comment:
In other components, the pattern is `alignIcon` and the text position is persistent.

Reply:
Updated